webapp/components/admin_console/email_settings.jsx

```diff
@@ -201,6 +203,25 @@ export default class EmailSettings extends AdminSettings {
                     onChange={this.handleChange}
                     disabled={!this.state.sendEmailNotifications}
                 />
+                <BooleanSetting
```

Comment:
Would you be okay moving the `Enable SMTP Authentication`, `SMTP Server Username` and `SMTP Server Password` settings below `SMTP Server Port`?

So the order of the settings would be 

```
SMTP Server
SMTP Server Port
Enable SMTP Authentication (true/false)
SMTP Server Username [disabled if auth = false]
SMTP Server Password [disabled if auth = false]
```

I think the flow is more intuitive since you first specify the SMTP server and port, then enable authentication (if you require it).